SaveData/DESCRIPTION

```diff
@@ -0,0 +1,22 @@
+Package: SaveData
+Title: Save and Summarize Data
+Version: 0.0.0.9000
+Authors@R: c(
+  person("Matthew", "Gentzkow", email = "gentzkow@stanford.edu", role = c("aut", "cre")), 
+  person("Jesse", "Shapiro", email = "jesse_shapiro_1@brown.edu", role = c("aut"))
+  )
+Description: Contains one function to save and summarize data. Use SaveData to save to standard formats and create a log file that summarizes the saved data. The log file gives standard summary statistics for numerical variables and displays variable types for all variables.
+Depends: 
+  R (>= 3.2.1)
+Imports:
+  data.table,
+  digest,
+  dplyr,
+  haven,
+  stargazer
+License: What license it uses
```

Comment:
Do we want to have a license for this package?

Reply:
Please ignore my previous comment. Now I found the license.

Reply:
Maybe it'd be good to direct to the license by saying something like: "Please refer to the LICENSE document."

Reply:
Done!